### ISSUE TITLE
Improve sound quality

### ISSUE THREAD

**Issue by darekmydlarz** [mortenjust/PocketCastsOSX#35]
Hey, podcasts played through app gives a very poor sound quality in comparison to playing via browser (Chrome in my case). What's the source of that problem, and how can it be solved?

**Comment by mortenjust** [comment]
How would you describe the difference in sound quality? This app uses Safari to run Pocket Casts, do you get equally bad quality in that browser?

**Comment by darekmydlarz** [comment]
Hey @mortenjust ! Thanks for follow up. Strange, I run app on other Macbook (company) and sounds good.
I will check in the afternoon on my private machine and let you know 👍

**Comment by darekmydlarz** [comment]
Hm.. interesting. On my private machine on Safari the sound quality is OK, but still at the app it's super poor. I have even restarted a computer, but didn't help.
As long as this is just my local stuff I think there is no problem in the app itself.

**Comment by ghost** [comment]
Yeah the app sounds horrible on my iMac. I use it in Chrome, and it sounds wayyyyyy better. So instead, I need to use Undercast to get a good experience with a native podcast app that isn't iTunes.

**Comment by darekmydlarz** [comment]
That's super strange, cause it works like a charm on another MacBook I own (company one).
It's super unclear for me what could be the cause of that.

**Comment by JulianBerger** [comment]
Same OS / Safari version?

**Comment by buffaloseven** [comment]
Are any people having this issue using a playback speed other than 1.0?
